src/create/projects/react-shadcn-spa/components.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
@shadcn let me know if you have any opinions on what we should put in the components.json `bun create` will inject when it detects shacdn